content/en/ref/release-notes/releases/0.71.md

```diff
@@ -72,6 +71,10 @@ With W&B Server 0.71, Registry is turned on by default for all organizations. Fo
 - Fixed wrapping of long names for Sweeps and Reports.
 - Improved the copy and paste behavior for console logs. Line numbers are no longer copied, and timestamps now appear inline instead of on a new line.
 
-<!-- ## Patches
+## Patches
 ### 0.71.1
-**August 6, 2025**-->
\ No newline at end of file
+**August 6, 2025**
+- Reverted the line plot behavior change announced in 0.71.0. Line plots with only 1 point for every line will convert to bar charts even if settings are changed.
+- Fixed a bug that could cause bounding boxes to render outside the media in a media panel instead of overlaying the media.
+- Weave traces submitted through Google Agent Development Kit or OpenTelemetry now translate the ADK format into a single JSON blob that includes both the request and response, and the chat view now displays both.

```

Comment:
```suggestion
- Fixed a Weave bug where Google Agent Development Kit traces submitted through OpenTelemetry displayed as JSON blobs instead of chat contents.
```